dietpi/dietpi-software

```diff
@@ -1421,6 +1421,13 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_DESC[$software_id]='validating, recursive, caching DNS resolver'
 		aSOFTWARE_CATX[$software_id]=13
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/dns_servers/#unbound'
+		
+		software_id=171
+
+		aSOFTWARE_NAME[$software_id]='AdGuardHome'
```

Comment:
Let's spell it "AdGuard Home" with the space, matching official spelling: https://adguard.com/de/adguard-home/overview.html

Reply:
done